### DISCUSSION TITLE
Mute audio button blocks back button in menus

### DISCUSSION THREAD

**Discussion by MulaHwaka** [ppy/osu#29344] [Ideas] [answered]
At the moment when chaning the volume in song select/results screen, the mute button blocks the back button in the bottom left (pictured). Having this button moved so that the back button is still fully accessible would be appreciated.
![image](https://github.com/user-attachments/assets/a9e9fab2-b961-47f4-991e-42098befce50)

**Comment by peppy** [comment]
As with most things, there's a new design for this already (well, at least for the volume controls, I haven't seen the mute button return). But until then, we can move it up alongside the controls.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/29346 mayb.
